components/mitsubishi_heatpump/espmhp.cpp

```diff
@@ -513,3 +515,16 @@ void MitsubishiHeatPump::dump_state() {
     LOG_CLIMATE("", "MitsubishiHeatPump Climate", this);
     ESP_LOGI(TAG, "HELLO");
 }
+
+void MitsubishiHeatPump::log_packet(byte* packet, unsigned int length, char* packetDirection) {
+    String packetHex;
+    char textBuf[15];
+
+    for (int i = 0; i < length; i++) {
+        memset(textBuf, 0, 15);
+        sprintf(textBuf, "%02X ", packet[i]);
+        packetHex += textBuf;
+    }
+    
+    ESP_LOGV(TAG, "PKT: [%s] %s", packetDirection, packetHex.c_str());
```

Comment:
Looks like ESP_LOGD would be the right function here, as we don't want this functionality to spam log messages unexpectedly.

Reply:
Can make that change - went with verbose to avoid flooding the debug log with things most people probably wouldn't care, but I'll push up a change.

Reply:
Typically DEBUG is more verbose that VERBOSE. But I just reviewed the Espressif documentation, and Verbose is more verbose that DEBUG. Weird.